[CI] Solaris 11x64 rsyslog failed — rsyslog solaris11x64 rsyslog — BUILD FAILED: failed 'gmake -j2 ...' (failure)

## Solaris 11x64 rsyslog CI failure

**Summary:** BUILD FAILED: failed 'gmake -j2 ...' (failure)
**Builder:** rsyslog solaris11x64 rsyslog
**Build:** https://build.rsyslog.com/#/builders/77/builds/7164
**Revision:** 3a02e69673b6bc9e47e557705de21eba813cd822
**Worker:** slave-solaris11x64

### Steps

- 0: worker_preparation — success

- 1: set http proxy in git config — success — [stdio](https://build.rsyslog.com/#/builders/77/builds/7164/steps/1/logs/stdio) 

- 2: git — success — [stdio](https://build.rsyslog.com/#/builders/77/builds/7164/steps/2/logs/stdio) 

- 3: shell — success — [stdio](https://build.rsyslog.com/#/builders/77/builds/7164/steps/3/logs/stdio) 

- 4: cleanup — success — [stdio](https://build.rsyslog.com/#/builders/77/builds/7164/steps/4/logs/stdio) 

- 5: cleanup dependencies — success — [stdio](https://build.rsyslog.com/#/builders/77/builds/7164/steps/5/logs/stdio) 

- 6: git branch information — success — [stdio](https://build.rsyslog.com/#/builders/77/builds/7164/steps/6/logs/stdio) 

- 7: building librelp dependency — success — [stdio](https://build.rsyslog.com/#/builders/77/builds/7164/steps/7/logs/stdio) 

- 8: building libfastjson dependency — success — [stdio](https://build.rsyslog.com/#/builders/77/builds/7164/steps/8/logs/stdio) 

- 9: autoreconf for SunStudio Compiler — success — [stdio](https://build.rsyslog.com/#/builders/77/builds/7164/steps/9/logs/stdio) 

- 10: shell_1 — success — [stdio](https://build.rsyslog.com/#/builders/77/builds/7164/steps/10/logs/stdio) [config.log](https://build.rsyslog.com/#/builders/77/builds/7164/steps/10/logs/config_log) 

- 11: build with SunStudio — success — [stdio](https://build.rsyslog.com/#/builders/77/builds/7164/steps/11/logs/stdio) 

- 12: shell_2 — failure — [stdio](https://build.rsyslog.com/#/builders/77/builds/7164/steps/12/logs/stdio) 

- 13: gathering make check logs — success — [stdio](https://build.rsyslog.com/#/builders/77/builds/7164/steps/13/logs/stdio) 

- 14: shell_3 — success — [stdio](https://build.rsyslog.com/#/builders/77/builds/7164/steps/14/logs/stdio)

--- a/tests/imtcp-stream-always-zlib-fragmented-ratio.sh
+++ b/tests/imtcp-stream-always-zlib-fragmented-ratio.sh
@@ -35,7 +35,8 @@ payload = "".join(
     "<129>Mar 10 01:00:00 172.20.245.8 tag: msgnum:%08d AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA\n" % i
     for i in range(0, 2000)
 ).encode("ascii")
-compressed = zlib.compress(payload, level=1)
+# Solaris' older Python accepts the compression level only positionally.
+compressed = zlib.compress(payload, 1)
 sock = socket.create_connection(("127.0.0.1", port))
 sock.setsockopt(socket.IPPROTO_TCP, socket.TCP_NODELAY, 1)
 for octet in compressed:

